tests/longevity-tests/jenkins-job.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+cd ~/vic
+
+git clean -fd
+git fetch https://github.com/vmware/vic master
+git pull
+
+cp ~/secrets .
+tests/longevity-tests/run-longevity.bash 6.0
```

Comment:
If I'm understanding correctly: should we also include the command for 6.5 in this script?

Reply:
yep, sorry meant to parameterize, should be good now